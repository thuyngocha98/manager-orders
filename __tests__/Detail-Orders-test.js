/**
 * @format
 */

import 'react-native';
import React from 'react';
import DetailOrder from '@modules/orders/DetailOrder';
// Note: test renderer must be required after react-native.
import * as renderer from 'react-test-renderer';

jest.mock('react-native-orientation-locker', () => {
  return {
    getInitialOrientation: jest.fn(),
  };
});

describe('render correctly', () => {
  const props = {
    route: {
      params: {nameProduct: 'S0N00113'},
    },
  };
  jest.useFakeTimers();
  it('render details orders', () => {
    const tree = renderer.create(<DetailOrder {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
