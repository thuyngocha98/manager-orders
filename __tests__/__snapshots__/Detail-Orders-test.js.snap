// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly render details orders 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#f0f3f8",
      "flex": 1,
      "flexDirection": "column",
    }
  }
>
  <Modal
    animationType="none"
    hardwareAccelerated={false}
    hideModalContentWhileAnimating={false}
    onModalHide={[Function]}
    onModalWillHide={[Function]}
    onModalWillShow={[Function]}
    onRequestClose={[Function]}
    scrollHorizontal={false}
    scrollOffset={0}
    scrollOffsetMax={0}
    scrollTo={null}
    statusBarTranslucent={true}
    supportedOrientations={
      Array [
        "portrait",
        "landscape",
      ]
    }
    swipeThreshold={100}
    transparent={true}
    visible={false}
  >
    <View
      accessible={true}
      focusable={true}
      forwardedRef={[Function]}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "black",
          "bottom": 0,
          "height": 1334,
          "left": 0,
          "opacity": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "width": 750,
        }
      }
    />
    <View
      forwardedRef={[Function]}
      hideModalContentWhileAnimating={false}
      onModalHide={[Function]}
      onModalWillHide={[Function]}
      onModalWillShow={[Function]}
      pointerEvents="box-none"
      scrollHorizontal={false}
      scrollOffset={0}
      scrollOffsetMax={0}
      scrollTo={null}
      statusBarTranslucent={true}
      style={
        Object {
          "bottom": 889.5,
          "flex": 1,
          "justifyContent": "center",
          "left": 0,
          "margin": 37.5,
          "marginHorizontal": 37,
          "position": "absolute",
          "right": 0,
          "transform": Array [
            Object {
              "translateY": 0,
            },
          ],
        }
      }
      supportedOrientations={
        Array [
          "portrait",
          "landscape",
        ]
      }
      swipeThreshold={100}
    >
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "borderRadius": 10,
            "flex": 1,
            "flexDirection": "column",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#0a1f44",
              "fontFamily": "HelveticaNeue-Medium",
              "fontSize": 60.5,
              "fontWeight": "normal",
              "marginVertical": 53.5,
              "textAlign": "center",
            }
          }
        >
          Ghi chú
        </Text>
        <View
          style={
            Object {
              "borderColor": "#98a1b1",
              "borderRadius": 5,
              "borderWidth": 1,
              "marginBottom": 53.5,
              "marginHorizontal": 14,
              "padding": 5,
            }
          }
        >
          <TextInput
            allowFontScaling={true}
            defaultValue=""
            multiline={true}
            numberOfLines={5}
            onChangeText={[Function]}
            rejectResponderTermination={true}
            style={
              Object {
                "color": "#0a1f44",
                "fontFamily": "HelveticaNeue-Medium",
                "fontSize": 60.5,
                "fontWeight": "normal",
                "height": 355.5,
                "paddingTop": 0,
                "textAlignVertical": "top",
              }
            }
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
        <View
          style={
            Object {
              "backgroundColor": "#98a1b1",
              "height": 1,
              "width": "100%",
            }
          }
        />
        <View
          style={
            Object {
              "alignItems": "center",
              "flex": 1,
              "flexDirection": "row",
              "marginHorizontal": 14,
            }
          }
        >
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "flexDirection": "column",
                "justifyContent": "center",
                "opacity": 1,
                "paddingVertical": 53.5,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#13AA52",
                  "fontFamily": "HelveticaNeue-Medium",
                  "fontSize": 53.5,
                  "fontWeight": "normal",
                }
              }
            >
              Thoát
            </Text>
          </View>
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "flexDirection": "column",
                "justifyContent": "center",
                "opacity": 1,
                "paddingVertical": 53.5,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#13AA52",
                  "fontFamily": "HelveticaNeue-Medium",
                  "fontSize": 53.5,
                  "fontWeight": "normal",
                }
              }
            >
              Xác nhận
            </Text>
          </View>
        </View>
      </View>
    </View>
  </Modal>
  <View
    style={
      Object {
        "backgroundColor": "#116149",
        "flexDirection": "column",
        "height": 226.5,
        "width": "100%",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
          "marginTop": 70,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "space-between",
            "marginHorizontal": 14,
          }
        }
      >
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <RNSVGSvgView
            align="xMidYMid"
            bbHeight={71}
            bbWidth={18.5}
            focusable={false}
            height={71}
            meetOrSlice={0}
            minX={0}
            minY={0}
            style={
              Array [
                Object {
                  "backgroundColor": "transparent",
                  "borderWidth": 0,
                },
                Object {
                  "flex": 0,
                  "height": 71,
                  "width": 18,
                },
              ]
            }
            vbHeight={447.243}
            vbWidth={447.243}
            width={18.5}
          >
            <RNSVGGroup>
              <RNSVGPath
                d="M420.361 192.229a31.967 31.967 0 00-5.535-.41H99.305l6.88-3.2a63.998 63.998 0 0018.08-12.8l88.48-88.48c11.653-11.124 13.611-29.019 4.64-42.4-10.441-14.259-30.464-17.355-44.724-6.914a32.018 32.018 0 00-3.276 2.754l-160 160c-12.504 12.49-12.515 32.751-.025 45.255l.025.025 160 160c12.514 12.479 32.775 12.451 45.255-.063a32.084 32.084 0 002.745-3.137c8.971-13.381 7.013-31.276-4.64-42.4l-88.32-88.64a64.002 64.002 0 00-16-11.68l-9.6-4.32h314.24c16.347.607 30.689-10.812 33.76-26.88 2.829-17.445-9.019-33.88-26.464-36.71z"
                fill={4294967295}
                propList={
                  Array [
                    "fill",
                  ]
                }
              />
            </RNSVGGroup>
          </RNSVGSvgView>
        </View>
        <Text
          style={
            Object {
              "color": "#FFFFFF",
              "fontFamily": "HelveticaNeue-Medium",
              "fontSize": 60.5,
              "fontWeight": "bold",
            }
          }
        >
          S0N00113
        </Text>
        <View
          style={
            Object {
              "width": 18.5,
            }
          }
        />
      </View>
    </View>
  </View>
  <RCTScrollView>
    <View>
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "paddingHorizontal": 14,
            "paddingVertical": 53.5,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#0a1f44",
              "fontFamily": "HelveticaNeue-Medium",
              "fontSize": 53.5,
              "fontWeight": "bold",
            }
          }
        >
          Tổng đơn hàng
        </Text>
        <Text
          style={
            Object {
              "color": "#0a1f44",
              "fontFamily": "HelveticaNeue-Medium",
              "fontSize": 71,
              "fontWeight": "bold",
              "textAlign": "center",
            }
          }
        >
          4.500.000 đ
        </Text>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#f0f3f8",
            "height": 35.5,
            "width": "100%",
          }
        }
      />
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "flexDirection": "column",
            "paddingHorizontal": 14,
            "paddingVertical": 53.5,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#0a1f44",
              "fontFamily": "HelveticaNeue-Medium",
              "fontSize": 53.5,
              "fontWeight": "bold",
            }
          }
        >
          Khách hàng
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "column",
                "marginTop": 18,
              }
            }
          >
            <Text
              numberOfLines={2}
              style={
                Object {
                  "color": "#0a1f44",
                  "fontFamily": "HelveticaNeue-Medium",
                  "fontSize": 46,
                  "fontWeight": "normal",
                  "marginTop": 18,
                }
              }
            >
              Tên khách hàng
              : 
              Nguyen Thi Kim Anh
            </Text>
            <Text
              numberOfLines={2}
              style={
                Object {
                  "color": "#0a1f44",
                  "fontFamily": "HelveticaNeue-Medium",
                  "fontSize": 46,
                  "fontWeight": "normal",
                  "marginTop": 18,
                }
              }
            >
              Số điện thoại
              : 
              0367451263
            </Text>
            <Text
              numberOfLines={2}
              style={
                Object {
                  "color": "#0a1f44",
                  "fontFamily": "HelveticaNeue-Medium",
                  "fontSize": 46,
                  "fontWeight": "normal",
                  "marginTop": 18,
                }
              }
            >
              Địa chỉ
              : 
              125 lý thường kiệt, quận 3, hồ chí minh
            </Text>
          </View>
          <View
            style={
              Object {
                "borderRadius": 106.5,
                "height": 213.5,
                "overflow": "hidden",
                "width": 213.5,
              }
            }
          >
            <Image
              source={
                Object {
                  "uri": "https://cdn.icon-icons.com/icons2/1378/PNG/512/avatardefault_92824.png",
                }
              }
              style={
                Object {
                  "height": 213.5,
                  "resizeMode": "cover",
                  "width": 213.5,
                }
              }
            />
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#f0f3f8",
            "height": 35.5,
            "width": "100%",
          }
        }
      />
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "flexDirection": "column",
            "paddingHorizontal": 14,
            "paddingVertical": 53.5,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#0a1f44",
              "fontFamily": "HelveticaNeue-Medium",
              "fontSize": 53.5,
              "fontWeight": "bold",
            }
          }
        >
          Sản phẩm
           (3)
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "borderBottomColor": "#e6e8ea",
              "borderBottomWidth": 1,
              "flexDirection": "row",
              "marginTop": 71,
              "paddingBottom": 28.5,
            }
          }
        >
          <View
            style={
              Object {
                "borderRadius": 8,
                "height": 213.5,
                "marginRight": 18.5,
                "width": 213.5,
              }
            }
          >
            <Image
              source={
                Object {
                  "uri": "https://cdn2.iconfinder.com/data/icons/metro-ui-icon-set/512/Image_-_Google_Docs.png",
                }
              }
              style={
                Object {
                  "borderRadius": 8,
                  "height": 213.5,
                  "resizeMode": "stretch",
                  "width": 213.5,
                }
              }
            />
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#FFFFFF",
                  "borderColor": "#13AA52",
                  "borderRadius": 39,
                  "borderWidth": 1,
                  "flexDirection": "column",
                  "height": 78.5,
                  "justifyContent": "center",
                  "position": "absolute",
                  "right": -39,
                  "top": -39,
                  "width": 78.5,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#13AA52",
                    "fontFamily": "HelveticaNeue-Medium",
                    "fontSize": 53.5,
                    "fontWeight": "bold",
                  }
                }
              >
                2
              </Text>
            </View>
          </View>
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "column",
                "justifyContent": "space-around",
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#0a1f44",
                    "fontFamily": "HelveticaNeue-Medium",
                    "fontSize": 53.5,
                    "fontWeight": "normal",
                  }
                }
              >
                Demo1
              </Text>
              <Text
                style={
                  Object {
                    "color": "#0a1f44",
                    "fontFamily": "HelveticaNeue-Medium",
                    "fontSize": 53.5,
                    "fontWeight": "normal",
                  }
                }
              >
                3.000.000 đ
              </Text>
            </View>
            <Text
              style={
                Object {
                  "color": "#98a1b1",
                  "fontFamily": "HelveticaNeue-Medium",
                  "fontSize": 46,
                  "fontWeight": "normal",
                }
              }
            >
              SKU: 
              PVN20
            </Text>
            <Text
              style={
                Object {
                  "color": "#98a1b1",
                  "fontFamily": "HelveticaNeue-Medium",
                  "fontSize": 46,
                  "fontWeight": "normal",
                }
              }
            >
              1.500.000 đ
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "borderBottomColor": "#e6e8ea",
              "borderBottomWidth": 1,
              "flexDirection": "row",
              "marginTop": 71,
              "paddingBottom": 28.5,
            }
          }
        >
          <View
            style={
              Object {
                "borderRadius": 8,
                "height": 213.5,
                "marginRight": 18.5,
                "width": 213.5,
              }
            }
          >
            <Image
              source={
                Object {
                  "uri": "https://cdn2.iconfinder.com/data/icons/metro-ui-icon-set/512/Image_-_Google_Docs.png",
                }
              }
              style={
                Object {
                  "borderRadius": 8,
                  "height": 213.5,
                  "resizeMode": "stretch",
                  "width": 213.5,
                }
              }
            />
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#FFFFFF",
                  "borderColor": "#13AA52",
                  "borderRadius": 39,
                  "borderWidth": 1,
                  "flexDirection": "column",
                  "height": 78.5,
                  "justifyContent": "center",
                  "position": "absolute",
                  "right": -39,
                  "top": -39,
                  "width": 78.5,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#13AA52",
                    "fontFamily": "HelveticaNeue-Medium",
                    "fontSize": 53.5,
                    "fontWeight": "bold",
                  }
                }
              >
                1
              </Text>
            </View>
          </View>
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "column",
                "justifyContent": "space-around",
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#0a1f44",
                    "fontFamily": "HelveticaNeue-Medium",
                    "fontSize": 53.5,
                    "fontWeight": "normal",
                  }
                }
              >
                Demo2
              </Text>
              <Text
                style={
                  Object {
                    "color": "#0a1f44",
                    "fontFamily": "HelveticaNeue-Medium",
                    "fontSize": 53.5,
                    "fontWeight": "normal",
                  }
                }
              >
                700.000 đ
              </Text>
            </View>
            <Text
              style={
                Object {
                  "color": "#98a1b1",
                  "fontFamily": "HelveticaNeue-Medium",
                  "fontSize": 46,
                  "fontWeight": "normal",
                }
              }
            >
              SKU: 
              PVN22
            </Text>
            <Text
              style={
                Object {
                  "color": "#98a1b1",
                  "fontFamily": "HelveticaNeue-Medium",
                  "fontSize": 46,
                  "fontWeight": "normal",
                }
              }
            >
              700.000 đ
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#f0f3f8",
            "height": 35.5,
            "width": "100%",
          }
        }
      />
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "flexDirection": "column",
            "paddingHorizontal": 14,
            "paddingVertical": 53.5,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "space-between",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#0a1f44",
                "fontFamily": "HelveticaNeue-Medium",
                "fontSize": 53.5,
                "fontWeight": "normal",
                "letterSpacing": 0.5,
                "marginBottom": 5,
              }
            }
          >
            Tổng tiền hàng
          </Text>
          <Text
            style={
              Object {
                "color": "#0a1f44",
                "fontFamily": "HelveticaNeue-Medium",
                "fontSize": 53.5,
                "fontWeight": "normal",
              }
            }
          >
            3.700.000 đ
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "space-between",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#0a1f44",
                "fontFamily": "HelveticaNeue-Medium",
                "fontSize": 53.5,
                "fontWeight": "normal",
                "letterSpacing": 0.5,
                "marginBottom": 5,
              }
            }
          >
            Phí giao hàng
          </Text>
          <Text
            style={
              Object {
                "color": "#0a1f44",
                "fontFamily": "HelveticaNeue-Medium",
                "fontSize": 53.5,
                "fontWeight": "normal",
              }
            }
          >
            800.000 đ
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "space-between",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#0a1f44",
                "fontFamily": "HelveticaNeue-Medium",
                "fontSize": 53.5,
                "fontWeight": "normal",
                "letterSpacing": 0.5,
                "marginBottom": 5,
              }
            }
          >
            Tiền khách phải trả
          </Text>
          <Text
            style={
              Object {
                "color": "#0a1f44",
                "fontFamily": "HelveticaNeue-Medium",
                "fontSize": 53.5,
                "fontWeight": "normal",
              }
            }
          >
            4.500.000 đ
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#f0f3f8",
            "height": 35.5,
            "width": "100%",
          }
        }
      />
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "flexDirection": "column",
            "marginBottom": 320,
            "paddingLeft": 14,
            "paddingRight": 9,
            "paddingVertical": 53.5,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "space-between",
            }
          }
        >
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#116149",
                  "fontFamily": "HelveticaNeue-Medium",
                  "fontSize": 53.5,
                  "fontWeight": "bold",
                }
              }
            >
              Thêm ghi chú
            </Text>
          </View>
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
                "paddingHorizontal": 4.5,
                "paddingVertical": 18,
              }
            }
          >
            <RNSVGSvgView
              align="xMidYMid"
              bbHeight={71}
              bbWidth={71}
              fill="#98a1b1"
              focusable={false}
              height={71}
              meetOrSlice={0}
              minX={0}
              minY={0}
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                    "borderWidth": 0,
                  },
                  Object {
                    "flex": 0,
                    "height": 71,
                    "width": 71,
                  },
                ]
              }
              vbHeight={24}
              vbWidth={24}
              width={71}
            >
              <RNSVGGroup
                fill={4288192945}
                propList={
                  Array [
                    "fill",
                  ]
                }
              >
                <RNSVGPath
                  d="M12.25 2h-1.1C10.82.85 9.76 0 8.5 0S6.18.85 5.85 2h-1.1c-.41 0-.75.34-.75.75v1.5C4 5.21 4.79 6 5.75 6h5.5C12.21 6 13 5.21 13 4.25v-1.5c0-.41-.34-.75-.75-.75z"
                  fill={4288192945}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M14.25 3H14v1.25C14 5.77 12.77 7 11.25 7h-5.5C4.23 7 3 5.77 3 4.25V3h-.25C1.23 3 0 4.23 0 5.75v12.5C0 19.77 1.23 21 2.75 21h7.38l.22-1.23c.1-.56.36-1.06.76-1.47l.8-.8H3.75c-.41 0-.75-.34-.75-.75s.34-.75.75-.75h9.5c.05 0 .09 0 .14.02h.01l3.6-3.6V5.75C17 4.23 15.77 3 14.25 3zm-1 11.25h-9.5c-.41 0-.75-.34-.75-.75s.34-.75.75-.75h9.5c.41 0 .75.34.75.75s-.34.75-.75.75zm0-3.25h-9.5c-.41 0-.75-.34-.75-.75s.34-.75.75-.75h9.5c.41 0 .75.34.75.75s-.34.75-.75.75zM12.527 24a.748.748 0 01-.738-.881l.53-3.005a.753.753 0 01.208-.4l7.425-7.424c.912-.914 1.808-.667 2.298-.177l1.237 1.237a1.75 1.75 0 010 2.475l-7.425 7.425a.739.739 0 01-.4.208l-3.005.53a.715.715 0 01-.13.012zm3.005-1.28h.01z"
                  fill={4288192945}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                />
              </RNSVGGroup>
            </RNSVGSvgView>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderColor": "#e6e8ea",
        "borderTopWidth": 2,
        "bottom": 0,
        "flexDirection": "row",
        "height": 284.5,
        "left": 0,
        "paddingBottom": 0,
        "position": "absolute",
        "right": 0,
        "width": "100%",
      }
    }
  >
    <View
      accessible={true}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#116149",
          "borderRadius": 5,
          "flex": 1,
          "flexDirection": "column",
          "height": 178,
          "justifyContent": "center",
          "marginLeft": 14,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#FFFFFF",
            "fontFamily": "HelveticaNeue-Medium",
            "fontSize": 57,
            "fontWeight": "bold",
          }
        }
      >
        Thanh toán
      </Text>
    </View>
    <View
      accessible={true}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#98a1b1",
          "borderRadius": 5,
          "flexDirection": "column",
          "height": 178,
          "justifyContent": "center",
          "marginLeft": 9,
          "marginRight": 14,
          "opacity": 1,
          "transform": Array [
            Object {
              "rotate": "90deg",
            },
          ],
          "width": 178,
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight={71}
        bbWidth={71}
        focusable={false}
        height={71}
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            Object {
              "flex": 0,
              "height": 71,
              "width": 71,
            },
          ]
        }
        vbHeight={60}
        vbWidth={60}
        width={71}
      >
        <RNSVGGroup>
          <RNSVGPath
            d="M30 16c4.411 0 8-3.589 8-8s-3.589-8-8-8-8 3.589-8 8 3.589 8 8 8zM30 44c-4.411 0-8 3.589-8 8s3.589 8 8 8 8-3.589 8-8-3.589-8-8-8zM30 22c-4.411 0-8 3.589-8 8s3.589 8 8 8 8-3.589 8-8-3.589-8-8-8z"
            fill={4294967295}
            propList={
              Array [
                "fill",
              ]
            }
          />
        </RNSVGGroup>
      </RNSVGSvgView>
    </View>
  </View>
</View>
`;
